### DISCUSSION TITLE
Transform shortcodes to Emoji in GitHub Actions

### DISCUSSION THREAD

**Discussion by JulianaGaibler** [community/community#9656] [Actions]
Hi!

I'm a big fan of using [Emojis](https://gitmoji.dev/) to annotate my commit messages. Recently I started using GitHub Actions and was sad to see that shortcodes are not transformed to emojis, like they do everywhere else on GitHub.

![Screenshot of GitHub Actions where shortcodes are written out instead of the associated Emojis](https://user-images.githubusercontent.com/16828024/148148584-3248809b-f412-4488-b9ce-f17e39b16c70.png)

It'd be great if the shortcodes would work for GitHub Actions too!

**Comment by chrispat** [comment]
Thank you for your feedback.  I agree this is something that should work the same across GitHub.  I will make sure to pass this along to the engineering team and see if we can get a quick fix.

**Reply by igotinfected** [reply]
Is this something that is still planned perhaps?

**Comment by JobaDiniz** [comment]
how did `moq` get it to work? https://github.com/moq/moq/runs/5087306364?check_suite_focus=true
![image](https://user-images.githubusercontent.com/165290/154331863-d81b11c2-fbee-4424-b1d5-569d633a62d3.png)

How can I use emoji like they did?

**Reply by suprstarrd** [reply]
They used the raw emoji. I hope I explain this well:
* When you type `:pray:`, it returns the character :pray:. 
* `:pray:` returns `🙏`, *not* `:pray:`.
* They used `🙏 restore` instead of `:pray: restore`.

**Reply by JobaDiniz** [reply]
Sorry, I don’t see the difference and I did nit quite understand your explanation

**Reply by suprstarrd** [reply]
The shortcode `:pray:` is different from the emoji character 🙏. The reason why the shortcode doesn't work is because Actions is only seeing the input `:pray:` as `:pray:` and not 🙏. `moq` got it to work because instead of inputting the shortcode that gives the emoji character (`:pray:` -> 🙏), they inputted just the emoji character directly (`🙏` -> 🙏).

**Reply by JobaDiniz** [reply]
I tried that as well, but the file won’t even save correctly.
Anyways, thanks for the explanation
